mods/fix-qt-scrollbar-grippers.wh.cpp

```diff
@@ -40,20 +40,56 @@ The mod will immediately apply to those programs after you click "Save".
 #include <windhawk_utils.h>
 #include <uxtheme.h>
 #include <vsstyle.h>
+#include <vector>
+#include <algorithm>
+
+std::vector<HTHEME> g_hScrollbarThemes;
+
+using OpenThemeData_t = decltype(&OpenThemeData);
+OpenThemeData_t OpenThemeData_orig;
+HTHEME OpenThemeData_hook(HWND hwnd, LPCWSTR pszClassList)
```

Comment:
Calling conventions are missing, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions

Reply:
This part has been rewritten.